GPU/Common/SoftwareTransformCommon.cpp

```diff
@@ -168,6 +168,13 @@ void SoftwareTransform::SetProjMatrix(float mtx[14], bool invertedX, bool invert
 	projMatrix_.translateAndScale(trans, scale);
 }
 
+static void ReadWeightedNormal(Vec3f &source, VertexReader &reader, u32 vertType) {
```

Comment:
I guess we can still have this func, but don't really need it now.  If we do keep, we should probably use it for the initial read too.

-[Unknown]